www/~/%username/subscriptions.json.spt

```diff
@@ -0,0 +1,41 @@
+""" Get or change authenticated user's subscriptions.
+"""
+from aspen import Response
+from gratipay.exceptions import NoTeam
+
+[------------------------------------------------------------------------]
+
+if user.ANON:
+    raise Response(403, _("Please sign in first."))
+
+else:
+    participant = user.participant
+
+    if request.method == 'GET':
+        slug = request.qs.get("team_slug", "")
+
+        if slug: # Get subscription info only for this team.
+            try:
+                subscription = participant.get_payment_instruction(slug)
+            except NoTeam:
+                raise Response(400, _("Invalid team name."))
+            out = {
+                    "amount" : str(subscription['amount']),
+                     "ctime" : subscription['ctime'],
+                     "mtime" : subscription['mtime']
+                  }
+        else: # Get subscription info for all teams.
+            subscriptions, totals = participant.get_giving_for_profile()
+
+            out = []
+            for s in subscriptions:
+                out.append({ "team_slug": s.team_slug,
+                             "team_name": s.team_name,
+                                "amount": str(s.amount),
```

Comment:
It was all over the code base. Eg., [this](https://github.com/gratipay/gratipay.com/blob/master/www/%25team/payment-instruction.json.spt) , [this](https://github.com/gratipay/gratipay.com/blob/1840/www/~/%25username/tip.json.spt) and [this](https://github.com/gratipay/gratipay.com/blob/1831/www/~/%25username/tips.json.spt). I was just being consistent. :stuck_out_tongue_closed_eyes: Figured it'd be some python thing.